course-v2/assets/css/course-v2.scss

```diff
@@ -167,6 +167,14 @@ pre {
   }
 }
 
+.touch-device-only {
+  display: none;
+
+  @media (pointer: coarse) {
+    display: block;
+  }
```

Comment:
Even though some devices have both, touch and mouse, this rule only checks the primary device. On PCs, this is a mouse. On tablets, this is touch. I tested this rule on a touch laptop and this stands true.

Reply:
Ok, the logic makes sense to me. I don't know if your PR reviewer will have a different opinion.